etl/steps/data/garden/insee/2024-03-21/inequality_france_1999.py

```diff
@@ -0,0 +1,32 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("inequality_france_1999")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["inequality_france_1999"].reset_index()
+
+    #
+    # Process data.
+    tb = tb.set_index(["country", "year"], verify_integrity=True)
```

Comment:
```suggestion
    tb = tb.format(["country", "year"])
```

Reply:
Yes, sorry, didn't re-checked this old PR. I will change this for all the cases I see